### DISCUSSION TITLE
Other methods to access/create SplitView

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [Floorp-Projects/Floorp#919] [Feature Request]
When using Sidebery, there are no way to create split view.

**Comment by CutterKnife** [comment]
`about:preferences#cSK` provides a way to open Split View using keyboard shortcuts.

**Reply by KAGEYAM4** [reply]
I looked at the settings twice.

Are you talking about sidebar? I meant SplitView, which is accessed by RightClickTab>FixedInSplitView>SplitRight.

**Reply by CutterKnife** [reply]
Split View is... Split View.
<img width="970" alt="スクリーンショット 2024-02-22 0 31 26" src="https://github.com/Floorp-Projects/Floorp/assets/110194318/593d3d11-264f-4863-8dfa-70ea10d0816d">

**Reply by KAGEYAM4** [reply]
Sorry, i was on 11.9.0, that's why this wasn't there. I updated now its there.

**Reply by KAGEYAM4** [reply]
@CutterKnife do you have anyidea for why is split is implemented this way? Like only pinning support because of this viminum and other keyboard bared extension only act on non pinned tab.

is is the limitation of the browser? If it can be made more UX friendly.

**Reply by CutterKnife** [reply]
I am not a developer, so I don't know what the implementation looks like.
But if there is something you think could be improved, you can create a new feature request or send a pull request to the repository.
